Main/game/game.js

```diff
@@ -146,24 +136,22 @@ export default class Game {
         const computerBid = document.getElementById('computer-bid');
         const showBidOptions = document.getElementById('bid-options');
 
-        const dJ = this.#players[1].name
-
-        setTimeout(function() {
-            showSelectedBid.innerHTML = `
-                <div class="selected-bid">
-                    <p>${dJ} bid </p>
-                    <span>${computerDiceCount} x </span>
-                    <img src="./../Images/Dice${computerDiceFace}.png"/>
-                </div> 
-            `;
-            computerBid.innerHTML = `
-                <button id="liar">Liar</button>
-            `;
-            showBidOptions.style.display = 'flex';
-        }, 3000)
-
-        this.showPlayerBidChoices();
+        const dJ = this.#players[1].name;
```

Comment:
```suggestion
        const { name } = this.#players[1].name;
```
why not just like this instead?

Reply:
ok will amend!